guardian/grid
# Upgrade dependencies in image-counter-lambda to fix build

## What does this change?

Update image-counter-lambda dependencies and node version to get it building again

Needs the lambda runtime also updating to v16

## How should a reviewer test this change?

Build and deploy to TEST and check that metrics are still being collected and stored.

## How can success be measured?

## Who should look at this?


## Tested? Documented?
